Sources/CodeEditTextView/CodeEditTextView.swift

```diff
@@ -26,6 +27,7 @@ public struct CodeEditTextView: NSViewControllerRepresentable {
         _ text: Binding<String>,
         language: CodeLanguage,
         theme: Binding<EditorTheme>,
+        useThemeBackground: Binding<Bool>,
```

Comment:
So we don't make a breaking change, can we move this to the end of the parameter list and give it a default value? Same with the other initializers.

This can also be a normal variable, no need for a binding I don't think.

Reply:
Yeah, I think as far as CETV is concerned, it should probably default to true. Does that work?